tests/unittests/tests-base64/tests-base64.c

```diff
@@ -303,13 +303,13 @@ static void test_base64_07_stream_decode(void)
 
 static void test_base64_08_encode_16_bytes(void)
 {
-    const int buffer_size = 16;
+    enum { buffer_size = 16 };
     unsigned char buffer[buffer_size];
     for (int i = 0; i < buffer_size; ++i) {
         buffer[i] = 'a';
     }
 
-    const size_t expected_out_size = 24;
```

Comment:
This one, too. Probably the compiler wouldn't use VLAs in here anyway, because it should be able to deduce that the length is constant, but if it does use VLAs then that it would be a need waste of stack and code size.